### ISSUE TITLE
Ensure site title and page titles match default pattern

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1059]
Themes from the Omeka Team should use the default pattern for site titles and page titles. Site titles should be strings wrapped in `<div class=“site-title”>` while page titles should be `h1` elements. This works towards heading level consistency with plugin content. The following themes are affected:

- [ ] Lively
- [ ] Rhythm

**Comment by kimisgold** [comment]
There should be h1s on the homepages of Rhythm, The Daily, Big Picture, Berlin, Center Row, Thanks Roy, Emiglio, Minimalist, and Santa Fe, all on their default branches.

**Comment by nelsonamaya82** [comment]
Adjusted Freedom and Lively accordingly.

**Comment by kimisgold** [comment]
@aviannamiller I think the themes should all be ready for a check.

**Comment by aviannamiller** [comment]
Homepages:
 
- The Daily
  - Skipped heading levels as there are no real section headers like in other themes. Ideally, whatever's currently tagged as h3 would be reassessed and converted to h2, if appropriate, to close that gap.
- Seasons
  - h1 is "Featured", even if featured records are not configured for that site's homepage.
- Foundation, Freedom
  - Following section headers on homepage are h2 but so are item links that should not be equal but instead be nested under the h2 in keeping with the visual structure.
- Rhythm
  - Missing h1
- Lively
  - h1 is unlabeled
- Freedom, Lively
  - If a banner is on the site, the h2 banner heading comes before the page h1. Is this an issue?
- Thanks, Roy
  - Two h1s, one the new addition of "Home", the other pulling in that "Featured" h1 above featured records. Same as Seasons, the "Featured" h1 remains even if records are not checked for display.

**Comment by kimisgold** [comment]
The Daily, Seasons, and Thanks Roy have their fixes in. Rhythm is using the core's default markup for the homepage, so the h1 has been added there on the `master` branch.

**Comment by aviannamiller** [comment]
Great, those look good.

Remaining issues involve Foundation, Freedom, and Lively.

**Comment by kimisgold** [comment]
Ah, I forgot to mention Foundation has a fix ready to test as well.

**Comment by aviannamiller** [comment]
Foundation is all set!

**Comment by nelsonamaya82** [comment]
@aviannamiller heading levels are fixed now, and I adjusted the banner heading so it doesn't affect the heading hierarchy for screen-reader or assistive-technology users.

**Comment by aviannamiller** [comment]
@nelsonamaya82 thanks for the changes to heading levels. Those look good, but issues remain with the banner headings. Now they're still styled as headings and we're calling them headings, yet programmatically, they're not.

On the front page the h1 could potentially relocate in relation to the banner, but this becomes a somewhat trickier issue on other pages with visible h1s that belong under the banner. @kimisgold, do you have any thoughts on a best approach here?

**Comment by kimisgold** [comment]
I'm wondering if that banner might make more sense marked up as an `<aside>`. It's not directly related to the main content, and would help us avoid multiple banner elements on the page. For the heading itself, an `<h2>` might make the most sense, as it still ranks lower than the page title in importance. That might be okay even if it's positioned before the page title; for instance, WAI documentation provides [an example of `<h1>` appearing after navigation](https://www.w3.org/WAI/tutorials/page-structure/headings/#main-heading-after-navigation). It's a fixed structural section, which I think gives us some leeway in its order.

**Comment by aviannamiller** [comment]
Sounds like a good plan to me!
